src/shop-app.html

```diff
@@ -250,6 +250,17 @@
         num-items="{{numItems}}"
         total="{{total}}"></shop-cart-data>
 
+    <!--
+      shop-payment-request provides the logic and hooks into the PaymentRequest API.
+    -->
+    <shop-payment-request
+        id="payment"
+        on-process-payment="_onProcessPayment"
```

Comment:
Minor nit: wdyt if we rename this event as `success`? My thinking is that it would be easier to  grasp since the user would know that this is a binary operation.  But feel free to keep it as it is.